sdk/identity/identity/src/client/errors.ts

```diff
@@ -164,7 +164,7 @@ export class AggregateAuthenticationError extends Error {
 
   constructor(errors: any[], errorMessage?: string) {
     const errorDetail = errors.join("\n");
-    super(`${errorMessage}\n\n${errorDetail}`);
+    super(`${errorMessage}\n${errorDetail}`);
```

Comment:
This meant a whole empty line, which I didn’t like now that I re-reviewed these errors.